src/olympia/devhub/views.py

```diff
@@ -1441,7 +1441,10 @@ def _submit_upload(request, addon, channel, next_view, wizard=False):
                 user=request.user,
             )
             version = addon.find_latest_version(channel=channel)
-            url_args = [addon.slug]
+            url_args = [
+                addon.slug,
+                'listed' if channel == amo.RELEASE_CHANNEL_LISTED else 'unlisted',
```

Comment:
you can use `amo.CHANNEL_CHOICES_API[channel]` to get the string from the channel constant

Reply:
I know, but that feels wrong - it's not an API value at all. That same logic is used in the templates and tests already.

Reply:
heh, we use the opposite constant, `CHANNEL_CHOICES_LOOKUP` throughout this file, it's just not got `API` in it's name.